packages/datastore/__tests__/helpers/UpdateSequenceHarness.ts

```diff
@@ -139,13 +146,13 @@ export class UpdateSequenceHarness {
 		this.isUserInputDelayed = true;
 	}
 
-	private isOutboxSettledAfterRevisions: boolean = false;
+	private isSettledAfterRevisions: boolean = false;
 
 	/**
 	 * Do we want to settle the outbox after each `Post` revision call?
```

Comment:
Since this now also awaits subscription responses, I'd probably update this comment:

```suggestion
	 * Determines whether we settle the outbox and await for the subscription response after each mutation.
```